plugins/org.jkiss.dbeaver.ext.altibase/plugin.xml

```diff
@@ -0,0 +1,278 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?eclipse version="3.2"?>
+<!--
+ * DBeaver - Universal Database Manager
+ * Copyright (C) 2010-2022 DBeaver Corp and others
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+  -->
+  
+<plugin>
+
+    <extension point="org.jkiss.dbeaver.generic.meta">
+        <meta id="alti" class="org.jkiss.dbeaver.ext.altibase.model.AltibaseMetaModel" driverClass="Altibase.jdbc.driver.AltibaseDriver,Altibase7_1.jdbc.driver.AltibaseDriver,Altibase7_3.jdbc.driver.AltibaseDriver"/>
+    </extension>
+
+    <extension point="org.jkiss.dbeaver.dataSourceProvider">
+        <datasource
+                class="org.jkiss.dbeaver.ext.altibase.AltibaseDataSourceProvider"
+                description="Altibase JDBC connector"
+                id="altibase"
+                parent="generic"
+                label="Altibase"
+                icon="icons/altibase_icon.png"
+                dialect="altibase">
+            <drivers managable="true">
+               <driver
+                     categories="sql"
+                     class="Altibase.jdbc.driver.AltibaseDriver"
+                     defaultDatabase="mydb"
+                     defaultPort="20300"
+                     defaultUser="SYS"
+                     description="Altibase JDBC Driver"
+                     icon="icons/altibase_icon.png"
+                     iconBig="icons/altibase_icon_big.png"
+                     id="Altibase"
+                     label="Altibase"
+                     sampleURL="jdbc:Altibase://{host}:{port}/{database}">
+                     
+                    <!-- Altibase requires JDBC files that match the version.
+                    <file type="jar" path="https://ftp.altibase.com/JDBC_Driver/Altibase-7.1.0.jar"/>

```

Comment:
We need a driver download link (from Maven or official vendor web site).
Unfortunately it is a showstopper for CE.

Reply:
Regarding the JDBC download link issue, I'll let you know as soon as I'm done. This issue will take some time, because the DB server and JDBC driver version compatibility policy and test automation method will be discussed and established.

Thanks for your comment on this issue.